client/lib/react-helpers/index.js

```diff
@@ -35,7 +36,11 @@ export function renderWithReduxStore( reactElement, domContainer, reduxStore ) {
 			: domContainer;
 
 	return ReactDom.render(
-		React.createElement( ReduxProvider, { store: reduxStore }, reactElement ),
+		<ReduxProvider store={ reduxStore }>
+			<Stylizer onInsertCss={ insertCss }>
```

Comment:
I don't think it makes a serious functional difference, but in `Layout` it would be a bit easier to discover.